backend/accounts/account.go

```diff
@@ -83,6 +85,9 @@ type Interface interface {
 	ProposeTxNote(string)
 	// SetTxNote sets a tx note and refreshes the account.
 	SetTxNote(txID string, note string) error
+
+	// ExportCSV exports the given transaction in CSV format (comma-separated).
+	ExportCSV(transactions []Transaction, w io.Writer) error
```

Comment:
Go typically takes output in the first argument. Would've been more consistent with stdlib:

```go
ExportCSV(io.Writer, []Transaction)
```

Reply:
Done